mk/config.mk

```diff
@@ -1284,3 +1284,9 @@ endif
 # handling of external abort implementing the plat_external_abort_handler()
 # function.
 CFG_EXTERNAL_ABORT_PLAT_HANDLER ?= n
+
+# CFG_CORE_LIBGCC, when enabled, links OP-TEE core and ldelf with libgcc.
+# Linking OP-TEE core and ldelf with libgcc is deprecated, but keep this
+# flag while sorting out remaining issues with removing the libgcc
+# dependency.
+CFG_CORE_LIBGCC ?= n
```

Comment:
What exactly needs `CFG_CORE_LIBGCC=y`? Perhaps we could avoid this CFG altogether?

Reply:
Nothing I'm aware of, I'm only trying to be nice to downstream for a while.

Reply:
That's good but tbh without the introduction of this CFG the patch changes only two lines so it's easy enough to revert.

Reply:
OK, I can remove it if that's preferred.